chapters/en/slides/module2/module2_02.Rmd

```diff
@@ -0,0 +1,279 @@
+---
+params:
+  dynamictitle: "module2_02"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Creating area charts
+
+Notes:
+In this module
+we will be learning how to create area charts 
+and when they are a suitable alternative to use
+instead of line charts.
+
+---
+
+## Global Development Data
+
+| Column                | Description                                                                                  |
+|-----------------------|----------------------------------------------------------------------------------------------|
+| country               | Country name                                                                                 |
+| year                  | Year of observation                                                                          |
+| population            | Population in the country at each year                                                       |
+| region                | Continent the country belongs to                                                             |
+| sub_region            | Sub-region the country belongs to                                                            |
+| income_group          | Income group [as specified by the world bank in 2018]                                        |
+| life_expectancy       | The mean number of years a newborn would <br>live if mortality patterns remained constant    |
+| income                | GDP per capita (in USD) <em>adjusted <br>for differences in purchasing power</em>            |
+| children_per_woman    | Average number of children born per woman                                                    |
+| child_mortality       | Deaths of children under 5 years <break>of age per 1000 live births                          |
+| pop_density           | Average number of people per km<sup>2</sup>                                                  |
+| co2_per_capita        | CO2 emissions from fossil fuels (tonnes per capita)                                          |
+| years_in_school_men   | Mean number of years in primary, secondary,<br>and tertiary school for 25-36 years old men   |
+| years_in_school_women | Mean number of years in primary, secondary,<br>and tertiary school for 25-36 years old women |
```

Comment:
This table is too long but it can be fixed by removing columns if we don't find a reticulate workaround. We can probably remove column anyways and maybe copy this data to a UBC repo (maybe this one?).